apollo/__main__.py

```diff
@@ -94,6 +95,8 @@ def add_dzhigurda_arg(my_parser):
     preprocessing_parser = subparsers.add_parser(
         "preprocess", help="Generate the dataset with good candidates for duplication.")
     preprocessing_parser.set_defaults(handler=preprocess_source)
+    preprocessing_parser.add_argument("-x", "--mode", choices=Moder.Options.__all__,
+                           default="file", help="What to select for analysis.")
```

Comment:
Looks like this is unaligned.

@eiso This is one of the examples of the future style checker patterns

Reply:
yes, it slipped below my radar